engines/pyfunc-ensembler-job/requirements.txt

```diff
@@ -1,8 +1,9 @@
+cloudpickle==2.0.0
 google-cloud-storage>=1.37.0
 jinjasql==0.1.8
-jinja2==3.0.3  # temporary fix to downgrade to Python 3.7
+jinja2==3.0.3
 mlflow==1.14.1
-numpy==1.17.3
```

Comment:
`numpy 1.17.3` is not available for Python 3.9 (See: https://anaconda.org/anaconda/numpy/files?version=1.17.3)